doc/source/Getting_started/Installation.rst

```diff
@@ -51,9 +51,9 @@ There are three options:
 
 - Custom PyAEDT scripts.
 
-- PyAEDT workflows. Existing workflows in the PyAEDT library.
+- Existing workflows in the PyAEDT library.
 
-- Open source PyAEDT toolkits. Open source PyAEDT toolkits following the `common library <https://aedt.common.toolkit.docs.pyansys.com/>`_ standard.
+- Open source PyAEDT toolkits described in the `PyAEDT Common Toolkit <https://aedt.common.toolkit.docs.pyansys.com/>`_ documentation.
```

Comment:
```suggestion
- Open source PyAEDT toolkits described in the `PyAEDT Common Toolkit documentation <https://aedt.common.toolkit.docs.pyansys.com/>`_.
```
Make "documentation part of the link text since it is where the link goes. (Otherwise, one might think it went to the repository for this library.)